test/Handler/SitesSpec.hs

```diff
@@ -108,3 +111,9 @@ spec = withApp $ do
 
 createSite_ :: UserId -> Site -> DB ()
 createSite_ userId = void . createSite userId
+
+selectLanguage :: Text -> RequestBuilder m ()
+selectLanguage lang = do
```

Comment:
Is there no general helper to select an option by label or value? We have to select by index?